MotDuJour/Project.swift

```diff
@@ -21,7 +21,7 @@ let infop: InfoPlist = .extendingDefault(with: [
 ])
 
 let p = Project(
-  name: "ComposableArchitecture",
+  name: "MotDuJour",
```

Comment:
The application is now named "Mot du Jour", and we have an internal library named "ComposableArchitecture."